File: ReportingApplication/ReportingApplication/ModelsDTO/Ticket/RecipentDTO.cs
using ReportingApplication.Models;

namespace ReportingApplication.ModelsDTO.Ticket
{
    public class RecipentDTO(string userId, string recipentName, string recipentColor)
    {
        public string? UserId { get; set; } = userId;
        public string? RecipentName { get; set; } = recipentName;
        public string? RecipentColor { get; set; } = recipentColor;
    }
}

File: ReportingApplicationFrontend/src/app/modelsDTO/recipentDTO.ts
export class RecipentDTO{
    id?: number | null;
    userId?: string | null;
    recipentName?: string | null;

    constructor(_id?: number | null, _userId?: string | null, _recipentName?: string | null){
        this.id = _id;
        this.userId = _userId;
        this.recipentName = _recipentName;
    }
}